modules/apps/admin-apps-page.php

```diff
@@ -46,7 +47,9 @@ private static function render_plugins_list() {
 	}
 
 	private static function get_plugins() : array {
-		if ( ! self::$wordpress_adapter ) {
+		if ( elementor_pro_container()->has( Wordpress_Adapter_Interface::class ) ) {
```

Comment:
Should core repo be aware of the pro repo?

Reply:
in general it's better to make PRO dependent on CORE